src/components/Layout/TopNav/TopNav.tsx

```diff
@@ -129,7 +129,7 @@ function NavItem({url, isActive, children}: any) {
           'active:scale-95 transition-transform w-full text-center outline-link py-1.5 px-1.5 xs:px-3 sm:px-4 rounded-full capitalize whitespace-nowrap',
           !isActive && 'hover:bg-primary/5 hover:dark:bg-primary-dark/5',
           isActive &&
-            'bg-highlight dark:bg-highlight-dark text-link dark:text-link-dark'
+            'bg-highlight dark:bg-highlight-dark text-link-10 dark:text-link-dark'
```

Comment:
I don't think `text-link-10` is set to anything? Can we use `test-link-secondary` from above?